Minifigures.UI/src/app/app.component.html

```diff
@@ -13,6 +16,9 @@
       <td>{{ boardgame.description }}</td>
       <td>{{ boardgame.avgPlaytime }}</td>
       <td>{{ boardgame.physicalMinis }}</td>
+      <td><button (click)="editBoardGame(boardgame)">Edit</button></td>
```

Comment:
:pick: Lets start using proper formating of html - if this button is inside of a td tag, then put it in a new line with a single tab